Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -860,6 +860,16 @@ extension HTTPClient.Configuration {
         /// - warning: Cycle detection will keep all visited URLs in memory which means a malicious server could use this as a denial-of-service vector.
         public static func follow(max: Int, allowCycles: Bool) -> RedirectConfiguration { return .init(configuration: .follow(max: max, allowCycles: allowCycles)) }
     }
+
+    /// Connection pool configuration.
+    public struct PoolConfiguration {
```

Comment:
Can we make this `Hashable`?

Reply:
Out of curiosity, why do you think we might need it? Also, `TimeAmount` is not hashable, so this will require implementing `hash(into:)`, should I add it?

Reply:
`TimeAmount` is `Hashable` as of 2.19.0.

Reply:
As to why we need it, it's not so much that we need it as that, for trivial structures, missing `Equatable` and `Hashable` conformances tend to be minor annoyances. It's nicer just to have them.

Reply:
done!